docs/v1beta_migration.md

```diff
@@ -0,0 +1,62 @@
+# Migrate to v1beta1
+
+OpenFeature Operator version [v0.5.0](https://github.com/open-feature/open-feature-operator/releases/tag/v0.5.0) contain improved, stable CRDs which were upgraded to `v1beta1` api version.
+This document shows what was changed and how you can upgrade your existing CRDs to be compatible with this new release.
+
+## Renaming FeatureFlagConfiguration to FeatureFlag
+
+Along with the API version upgrade to `v1beta1`, we have renamed `FeatureFlagConfiguration` to `FeatureFlag`, making the naming clearer for the CRD maintainers. 
```

Comment:
```suggestion
Along with the API version upgrade to `v1beta1`, we have renamed `FeatureFlagConfiguration` to `FeatureFlag`, making the naming clearer for the custom resource maintainers. 
```

I think the CRDs are maintained by us. I think it's the CRs (instances) that you mean here.